Sources/MusicXML/Complex Types/Note.swift

```diff
@@ -150,40 +139,39 @@ extension Note: Codable {
         self.notations = try container.decodeIfPresent(Notations.self, forKey: .notations)
         self.lyrics = try container.decodeIfPresent([Lyric].self, forKey: .lyrics)
         self.play = try container.decodeIfPresent(Play.self, forKey: .play)
+        let chordEmptyElement = try container.decodeIfPresent(Empty.self, forKey: .chord)
```

Comment:
This is interesting. Have you tried reenabling the test `NoteTests.DISABLED_testChord()`? We have been having `<chord/>` issues, but perhaps it was user error on my side?

cc @bwetherfield.

Reply:
`testChord` errors.